doc-content/jbpm-docs/src/main/asciidoc/Integration/integration-maven-dependency-con.adoc

```diff
@@ -0,0 +1,160 @@
+[id='integration-maven-dependency-con_{context}']
+= Dependency management with Maven
+
+When you build projects that embed the {PROCESS_ENGINE}, use Apache Maven to configure all dependencies required by the {PROCESS_ENGINE}.
+
+The {PROCESS_ENGINE} provides a set of BOMs (Bill Of Material) to simplify declaring artifact dependencies.
+
+Use the top-level `pom.xml` file of your project to define dependency management for embedding the {PROCESS_ENGINE}, as shown in the following example. The example incluides the main runtime dependencies, which are applicable  whether the application is deployed on an application server, in a servlet container, or as a stand-alone application.
+
+.Maven dependency management settings for embedding the {PROCESS_ENGINE}
+[source,xml]
```

Comment:
Be sure that any snippets like this one set the source with the `attributes+` tag to render the variable in the snippet, otherwise just shows `{COMMUNITY_VERSION_FINAL}` literally in the output. And the `+quotes` tag for rendering ital properly with placeholder text, like `__USER_NAME__`, etc. (not applicable here, just mentioning).

```
[source,xml,subs="attributes+,+quotes"]
```